package.json should be updated to support TypeScripts bundler module resolution

I am trying to use TypeScripts `"moduleResolution": "Bundler"`-feature and I'm blocked by the `sass-embedded`-package.

This is the error that I'm getting from TypeScript:
> buildscript.ts:1:30 - error TS7016: Could not find a declaration file for module 'sass-embedded'. '/projectPath/node_modules/sass-embedded/dist/lib/index.mjs' implicitly has an 'any' type.
>   There are types at '/projectPath/node_modules/sass-embedded/dist/types/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'sass-embedded' library may need to update its package.json or typings.

From what I can gather it seems that it is no longer enough to specify `"types": "dist/types/index.d.ts"`, the package file should also include the types in exports as well.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sass-embedded",
-  "version": "1.67.0",
+  "version": "1.67.1-dev",
   "protocol-version": "2.1.0",
   "compiler-version": "1.67.0",
   "description": "Node.js library that communicates with Embedded Dart Sass using the Embedded Sass protocol",
@@ -8,7 +8,11 @@
   "author": "Google Inc.",
   "license": "MIT",
   "exports": {
-    "import": "./dist/lib/index.mjs",
+    "import": {
+      "types": "./dist/lib/index.m.d.ts",
+      "default": "./dist/lib/index.mjs"
+    },
+    "types": "./dist/lib/index.d.ts",
     "default": "./dist/lib/index.js"
   },
   "main": "dist/lib/index.js",
@@ -25,7 +29,7 @@
     "check:gts": "gts check",
     "check:tsc": "tsc --noEmit",
     "clean": "gts clean",
-    "compile": "tsc && cp lib/index.mjs dist/lib/index.mjs",
+    "compile": "tsc && cp lib/index.mjs dist/lib/index.mjs && cp -r lib/src/vendor/sass/ dist/lib/src/vendor/sass && cp dist/lib/src/vendor/sass/index.d.ts dist/lib/src/vendor/sass/index.m.d.ts",
     "fix": "gts fix",
     "prepublishOnly": "npm run clean && ts-node ./tool/prepare-release.ts",
     "test": "jest"

